packages/ui-service-inquirer/src/translations/ui-service-inquirer.i18n.json

```
@@ -0,0 +1,18 @@
+{
+    "MESSAGE_OBJECT_TYPE": "Object Type",
+    "MESSAGE_BUSINESS_OBJECT_INTERFACE": "Business Object Interface",
+    "MESSAGE_ABAP_CDS_SERVICE": "ABAP Core Data Service",
+    "PROMPT_DRAFT_ENABLED": "Draft Enabled",
+    "PROMPT_LAUNCH_APP_GEN": "Do you want to create an SAP Fiori application with the newly generated service?",
+    "PROMPT_LAUNCH_APP_GEN_BREADCRUMB": "Create SAP Fiori Application?",
+    "ERROR_VALIDATING_CONTENT": "There was a backend validation error for the service content and package provided, service generation cannot continue.",
+    "ERROR_FETCHING_CONTENT_FOR_SERVICE_BINDING": "Error fetching content for business object service binding",
```

```suggestion
    "ERROR_FETCHING_CONTENT_FOR_SERVICE_BINDING": "Error fetching content for the service binding of the business object service",
```
What advice can we give to users to fix this? Should they try again?

---

Thanks I've made the suggested change. Also added `Please try again or contact an admin for the backend system.` to the message, thats all we can do for the service generator flow